src/Microsoft.AspNet.Hosting/Startup/ConfigureServicesDelegate.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using Microsoft.AspNet.Builder;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.Hosting.Startup
+{
+
+    public class ConfigureServicesDelegate
+    {
+        public ConfigureServicesDelegate(MethodInfo configureServices)
+        {
+            MethodInfo = configureServices;
+        }
+
+        public MethodInfo MethodInfo { get; }
+
+        public IServiceProvider Invoke(object instance, IApplicationBuilder builder, IServiceCollection services)
```

Comment:
I think the IApplicationBuilders being passed around are for the static versions of ConfigureServices/Configure